docs/source/usage/parameters.rst

```diff
@@ -261,59 +261,64 @@ Lattice Elements
          * ``cfbend`` for a combined function bending magnet. This requires these additional parameters:
 
             * ``<element_name>.ds`` (``float``, in meters) the segment length
-
             * ``<element_name>.rc`` (``float``, in meters) the bend radius
-
             * ``<element_name>.k`` (``float``, in inverse meters squared) the quadrupole strength
-
                 = (magnetic field gradient in T/m) / (magnetic rigidity in T-m)
 
               * k > 0 horizontal focusing
               * k < 0 horizontal defocusing
 
+            * ``<element_name>.dx`` (``float``, in meters) horizontal translation error
+            * ``<element_name>.dy`` (``float``, in meters) vertical translation error
+            * ``<element_name>.rotation`` (``float``, in degrees) transversal rotation error around the z axis (in the x-y plane)
```

Comment:
```suggestion
            * ``<element_name>.rotation`` (``float``, in degrees) rotation error around the z axis (in the x-y plane)
```

Reply:
"Transverse" is redundant here, since z is given as the rotation axis (and the x-y plane is also specified).
If it appears, recommend "transversal" -> "transverse".

Reply:
Should we also write `s` axis instead of `z` axis?

Reply:
Hm.  Since `s` denotes the path length of the nominal reference orbit, it doesn't really coincide with an axis.  However, using `z` could also be confusing.  We could also say "rotation error around the magnetic axis".  For straight elements, this is a well-defined property of the magnet.

Reply:
Or we briefly say `rotation  error in the x-y plane`?

Reply:
Discussed: `rotation error in the transverse plane`